src/styles/utils/_normalaize.scss

```diff
@@ -0,0 +1,4 @@
+body {
```

Comment:
The file name seems to have a typo. It should be '_normalize.scss' instead of '_normalaize.scss'. Please rename the file to avoid confusion.